### ISSUE TITLE
Problem with converting a Julia Array to a Numpy ndarray

### ISSUE THREAD

**Issue by simulkade** [stevengj/PyCall.jl#128]
I'm using smooth function from scipy cookbook, which used to work fine for me until today that I got this error:
```
using PyCall
@pyimport smooth
a=rand(50)
smooth.smooth(a)

ERROR: PyError (:PyObject_Call) <type 'exceptions.AttributeError'>
AttributeError("'list' object has no attribute 'ndim'",)
  File "/usr/lib/python2.7/dist-packages/smooth.py", line 42, in smooth
    if x.ndim != 1:
```
Then I tried to see if PyCall converts the array correctly by running
```
a=rand(5)
b=PyObject(a)
b[:ndim]
```
which returns
```
ERROR: key not found: "ndim"
 in getindex at /home/ali/.julia/v0.3/PyCall/src/PyCall.jl:252
 in getindex at /home/ali/.julia/v0.3/PyCall/src/PyCall.jl:257
```
Any suggestions?

**Comment by stevengj** [comment]
Should be fixed by 05073867acaff4ea2514e3c988e79ffce54cbfbb

**Comment by simulkade** [comment]
Thanks a lot Steven. It works now.
